src/routes/auth.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
```

Comment:
[Naming]
Again here I'd call this the User router not the auth router unless you want to separate the two as we did in week 11 of the class. In that case we'd need to create 2, but personally I see no reason to do that right now.

Reply:
I renamed auth with User as recommended.